Tests/Shared.Specs/TypeAssertionSpecs.cs

```diff
@@ -484,6 +484,65 @@ public void When_an_unrelated_type_instance_it_fails_with_a_useful_message()
                 .WithMessage($"*{typeof(DummyImplementingClass)} to be assignable to {typeof(DateTime)}*failure message*");
         }
 
+        [Fact]
+        public void When_constructed_of_open_generic_it_succeeds()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange / Act / Assert
+            //-----------------------------------------------------------------------------------------------------------
+            typeof(IDummyInterface<IDummyInterface>).Should().BeAssignableTo(typeof(IDummyInterface<>));
+        }
+
+        [Fact]
+        public void When_implementation_of_open_generic_interface_it_succeeds()
```

Comment:
🙃 It feels like the name is missing a verb or something.

Reply:
I was trying to emulate the "When_its_own_type_instance_it_succeeds" pattern the other BeAssignableTo methods were following without making the names super long.